### PR TITLE
Add support for set values

### PR DESCRIPTION
**Pull request by ThePletch**
Introduces simple merge strategies for sets, defaulting to merging them by union.

### CODE REVIEW

**Review by toumorokoshi — CHANGES_REQUESTED** [review]
Thanks for the PR! The overall change makes sense. A couple notes:

- looks like the set.py file was not added, which is also causing the CI to fail. could you add that in?
- could you add unit tests as to the expected behavior of set merging, as well as a unit test to verify the behavior of  the default mergers?

Thanks again!

File: deepmerge/__init__.py [hunk 1]
```diff
@@ -1,17 +1,19 @@
 from .merger import Merger
-from .strategy.core import STRATEGY_END
```
Comment (ThePletch) [inline deepmerge/__init__.py:2]: oh huh, how'd this get deleted?

**Comment by ThePletch** [comment]
is it alright if i drop the python 2.7 and pypy builds from the travis config, or are you still looking to maintain python 2.7 support? it looks like they're failing due to unrelated dependency issues.

**Comment by toumorokoshi** [comment]
> is it alright if i drop the python 2.7 and pypy builds from the travis config, or are you still looking to maintain python 2.7 support? it looks like they're failing due to unrelated dependency issues.

I think that's fine, although that may also require changing the pypy env to pypy3. 

If it's fairly clear to you on how to make that change, go for it. I might just try it now too.

**Comment by toumorokoshi** [comment]
> > is it alright if i drop the python 2.7 and pypy builds from the travis config, or are you still looking to maintain python 2.7 support? it looks like they're failing due to unrelated dependency issues.
> 
> I think that's fine, although that may also require changing the pypy env to pypy3.
> 
> If it's fairly clear to you on how to make that change, go for it. I might just try it now too.

okay fixed in #12. If you rebase it should it fix your build.

**Comment by toumorokoshi** [comment]
LGTM, thanks!

**Comment by toumorokoshi** [comment]
shipped as [0.3.0](https://pypi.org/project/deepmerge/0.3.0/), thanks again!

**Pull request merged**